src/main/resources/assets/pokecube_mobs/gen_5/entity/models/emolga.xml

```diff
@@ -3,10 +3,11 @@
  <model>
   <phase name="global" offset="0,0,0" rotation="1,0,0,0" scale="0.3"/>
   <metadata head="head" headAxis="2" headAxis2="0" headCap="-80,80" headCap1="-50,50" headDir="-1"/>
-  <customTex default="emolga">

```

Comment:
You probably don't want to remove these 4 lines (they were added since you started editing your end)